code/style.css

```diff
@@ -1,7 +1,214 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
I don't know if you have read about the box-sizing property (don't think we discussed it in the classroom..). I find it really usefull to apply to my projects to ensure that width + padding + border and height of your divs are the same. Hard to explain but here is a great example from w3 schools and MDN : 
https://www.w3schools.com/css/css3_box-sizing.asp
https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing

In this case I don't believe it was necessary for you to have it in your code but wanted to give you the tip if you have not red about it! For future projects :D